doc/api/process.md

```diff
@@ -472,8 +472,8 @@ added: 6.4.0
 
 * {string}
 
-The `process.argv0` property stores a read-only copy of the original value of
-`argv[0]` passed when Node.js starts.
+`process.argv0` is a configurable property that stores a read-only copy of the
```

Comment:
IMO "configurable property" lays way too much focus on the fact that it is configurable. Many users might not be too familiar with the concept of ES5 properties. I would prefer a subtle additional sentence such as "Note that this property is configurable and therefore can be reconfigured, effectively making it writable" (even though this phrase might not be good either).